splice_spark2/README.md

```diff
@@ -0,0 +1,15 @@
+## Native Spark Data Source (NSDS)
+
+### Troubleshooting
+
+#### Kafka Bootstrap Server Config
+
+NSDS times out on Kafka when this param isn't set properly.
+The _splice.kafka.bootstrapServers_ param is used by the classes in the DB to connect to Kafka.
+
+In the colos, be sure the config parameter _splice.kafka.bootstrapServers_ is set as described in 
```

Comment:
this documentaion is visible to customers and open source users.  Please uplevel these comments so that it doesn't refer to COLO and cloud.  Instead refer to bare metal and bespoke, versus cloud and Kubernetes installations.

With that change, this PR can be merged.